### PR TITLE
[#252] Enable user stamps for tally sheet lock and submit

### LINKED ISSUE #252: Enable user stamps for lock, unlock etc.

### PR DESCRIPTION
**Pull request by callkalpa**
Fixes #252

### CODE REVIEW

File: orm/entities/Submission/__init__.py [hunk 1]
```diff
@@ -15,7 +16,9 @@ class SubmissionModel(db.Model):
     submissionProofId = db.Column(db.Integer, db.ForeignKey(Proof.Model.__table__.c.proofId), nullable=False)
     latestVersionId = db.Column(db.Integer, db.ForeignKey("submissionVersion.submissionVersionId"), nullable=True)
     lockedVersionId = db.Column(db.Integer, db.ForeignKey("submissionVersion.submissionVersionId"), nullable=True)
```
Comment (dinukadesilva) [inline orm/entities/Submission/__init__.py:19]: Shall we add the this to the latest version too..

**Review by dinukadesilva — CHANGES_REQUESTED** [review]

File: orm/entities/Submission/__init__.py (hunk 1 above)
Comment (callkalpa) [reply orm/entities/Submission/__init__.py:19]: This is done.

File: schemas/__init__.py [hunk 2]
```diff
@@ -23,6 +24,21 @@
 from marshmallow_enum import EnumField
 
 
+class StampSchema(ma.ModelSchema):
+    class Meta:
+        fields = (
+            "stampId",
+            "ip",
```
Comment (dinukadesilva) [inline schemas/__init__.py]: Lets remove the "ip"

**Review by dinukadesilva — APPROVED** [review]

File: schemas/__init__.py (hunk 2 above)
Comment (callkalpa) [reply schemas/__init__.py]: Done.

**Pull request merged**